NOTES-NONSTOP.md

```diff
@@ -0,0 +1,155 @@
+    the `--with-rand-seed=egd` option to the `./Configure` script.
+    
+
+About C99 compiler
+------------------
+
+ The c99 compiler is required for building OpenSSL from source. While c11
+ may work, it has not been broadly tested. c99 is the only compiler
+ prerequisite needed to build OpenSSL 3.0 on this platform. You should also
+ have the FLOSS package installed on your system. The ITUGLIB FLOSS package
+ is the only FLOSS variant that has been broadly tested.
+ 
+About Prefix and OpenSSLDir
+---------------------------
+
+ Because there are many potential builds that must co-exist on any given
+ NonStop node, managing the location of your build distribution is crucial.
+ Keep each destination separate and distinct. Mixing any mode described in
+ this document can cause application instability. The recommended approach
+ is to specify the OpenSSL version and threading model in your configuration
+ options, and keeping your memory and float options consistent, for example:
+ 
+ * For 1.1 `--prefix=/usr/local-ssl1.1 --openssldir=/usr/local-ssl1.1/ssl` 
+ * For 1.1 PUT `--prefix=/usr/local-ssl1.1_put --openssldir=/usr/local-ssl1.1_put/ssl`
+ * For 3.0 `--prefix=/usr/local-ssl3.0 --openssldir=/usr/local-ssl3.0/ssl` 
+ * For 3.0 PUT `--prefix=/usr/local-ssl3.0_put --openssldir=/usr/local-ssl3.0_put/ssl`
+
+ Use the `_RLD_LIB_PATH` environment variable in OSS to select the appropriate
+ directory containing `libcrypto.so` and `libssl.so`. In GUARDIAN, use the
+ `=_RLD_LIB_PATH` search define to locate the GUARDIAN subvolume where OpenSSL
+ is installed.
+
+Threading Models
+----------------
+
+ OpenSSL can be built using unthreaded, POSIX User Threads (PUT), or Standard
+ POSIX Threads (SPT). Select the following build configuration for each on
+ the TNS/X (L-Series) platform:
+ 
+ * `nonstop-nsx` or default will select an unthreaded build.
+ * `nonstop-nsx_put` selects the PUT build.
+ * `nonstop-nsx_64_put` selects the 64 bit file length PUT build.
+ * `nonstop-nsx_spt_floss` selects the SPT build with FLOSS. FLOSS is
+   required for SPT builds because of a known hang when using SPT on its own.
+ 
+### TNS/E Considerations
+   
+ The TNS/E platform is build using the same set of builds specifying `nse`
+ instead of `nsx` in the set above.
+ 
+ You cannot build for TNS/E for FIPS, so you must specify the `no-fips`
+ option to `./Configure`
+ 
+Float Considerations
+--------------------
+
+ OpenSSL is built using IEEE float mode by default. If you need a different
+ IEEE mode, create a new configuration specifying `tfloat-x86-64` (for Tandem
+ float) or `nfloat-x86-64` (for Neutral Float).
+ 
+Memory Models
+-------------
+
+ The current OpenSSL default memory model uses the default platform address
+ model. If you need a different address model, you must specify the appropriate
+ c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
+
+Dynamic Engines
+---------------
+
```

Comment:
Something related to providers should be said here. And as the engines are deprecated, I think it should be noticed here too.

Reply:
I removed the Dynamic Engines references. There's not really much sense duplicating documentation here on the subject.